### ISSUE TITLE
CI for rpmlint is broken

### ISSUE THREAD

**Issue by ledif** [ublue-os/packages#75]
## Problem

Any commit that touches an rpm spec file will have a red CI because the rpmlint action doesn't take into account rpkg specific macro expansions such as `{{ git_dir_version }}`.

For example, see https://github.com/ublue-os/packages/actions/runs/12488530958/job/34851009173:
```bash
framework-laptop-kmod-common.spec: E: specfile-error error: line 4: Tag takes single token only: Version:        {{{ git_dir_version }}}
framework-laptop-kmod-common.spec: E: specfile-error error: query of specfile akmods/framework-laptop/framework-laptop-kmod-common.spec failed, can't parse
framework-laptop-kmod-common.spec: E: specfile-error can't parse specfile akmods/framework-laptop/framework-laptop-kmod-common.spec
framework-laptop-kmod.spec: E: specfile-error error: line 9: Tag takes single token only: Version:        {{{ git_dir_version }}}
framework-laptop-kmod.spec: E: specfile-error error: query of specfile akmods/framework-laptop/framework-laptop-kmod.spec failed, can't parse
framework-laptop-kmod.spec: E: specfile-error can't parse specfile akmods/framework-laptop/framework-laptop-kmod.spec
```

### 1. Implement support for rpkg macros in [EyeCantCU/rpmlint-action](https://github.com/EyeCantCU/rpmlint-action)

I started on this path with a fork ([ledif/rpmlint-action](https://github.com/EyeCantCU/rpmlint-action/compare/main...ledif:rpmlint-action:main)) and was able to enable linting for these packages by using `rpkg lint` instead of `rpmlint`. The reporting is different in terms of its verbosity, but it is seemingly kind of working.

I can continue on this path if there is interest by:
1. Finalizing the changes and submitting a PR to [EyeCantCU/rpmlint-action](https://github.com/EyeCantCU/rpmlint-action).
2. Update the actions in this repo to use the updated version with this functionality when the PR is merged.

I have lost a bit of steam on this front, but I can pick it up if there is interest.

### 2. Remove the linting workflow

We can also just disable this workflow since it might not be providing as much value currently in its broken state.

**Comment by castrojo** [comment]
Yeah we should probably just turn it off unless there's some other linting action that people are using?

**Comment by ledif** [comment]
Resolved in #76 by removing the job.
